### DISCUSSION TITLE
texture to base 64 auto triggering on dev

### DISCUSSION THREAD

**Discussion by marioepsley** [cables-gl/cables_docs#690] [! Welcome - readme first!]
On dev I'm finding the texturetobase64 op auto triggering

<img width="165" alt="Screenshot 2024-02-14 at 11 40 45" src="https://github.com/cables-gl/cables_docs/assets/36418408/93165b47-8f46-4068-96a1-c09550dc5ea7">

keeps updating my sidebar image even when i'm not triggering it

**Comment by steam0r** [comment]
can you create an example patch for this? are you changing the texture?
